website/source/guides/packer-on-cicd/building-virtualbox-image.html.md

```diff
@@ -0,0 +1,102 @@
+provider "packet" { }
+
+resource "packet_project" "teamcity_agents" {
+  name = "TeamCity"
+}
+
+resource "packet_device" "agent" {
+  hostname         = "teamcity-agent"
+  plan             = "baremetal_0"
+  facility         = "ams1"
+  operating_system = "ubuntu_16_04"
+  billing_cycle    = "hourly"
+  project_id       = "${packet_project.teamcity_project.id}"
+}
+```
+
+## 2. Install VirtualBox and TeamCity Dependencies
+
+VirtualBox must be installed on the new instance along and TeamCity requires the JDK prior to installation. This guide uses Ubuntu as the Linux distribution, so you may need to adjust these commands for your distribution of choice.
+
+**Install Teamcity Dependencies**
+
+```shell
+apt-get upgrade
+apt-get install -y zip linux-headers-generic linux-headers-4.13.0-16-generic build-essential openjdk-8-jdk
+```
+
+**Install VirtualBox**
+
+```
+curl -OL "http://download.virtualbox.org/virtualbox/5.2.2/virtualbox-5.2_5.2.2-119230~Ubuntu~xenial_amd64.deb"
+dpkg -i virtualbox-5.2_5.2.2-119230~Ubuntu~xenial_amd64.deb
+```
+
+You can also use the [`remote-exec` provisioner](https://www.terraform.io/docs/provisioners/remote-exec.html) in your terraform configuration to automatically run these commands when provisioning the new instance.
+
+## 3. Install Packer
+
+The TeamCity Agent machine will also need Packer Installed. You can find the latest download link from the [Packer Download](https://www.packer.io/downloads.html) page.
+
+```shell
+curl -OL "https://releases.hashicorp.com/packer/1.1.2/packer_1.1.2_linux_amd64.zip"
+unzip ./packer_1.1.2_linux_amd64.zip
+```
+
+Packer is installed at the `/root/packer` path which is used in subsequent steps. If it is installed elsewhere, take note of the path.
+
+## 4. Install TeamCity Agent
+
+This guide assume you already have a running instance of TeamCity Server. The new TeamCity Agent can be installed by [downloading a zip file and installing manually](https://confluence.jetbrains.com/display/TCD10//Setting+up+and+Running+Additional+Build+Agents#SettingupandRunningAdditionalBuildAgents-InstallingAdditionalBuildAgents), or using [Agent Push](https://confluence.jetbrains.com/display/TCD10//Setting+up+and+Running+Additional+Build+Agents#SettingupandRunningAdditionalBuildAgents-InstallingviaAgentPush). Once it is installed it should appear in TeamCity as a new Agent. 
+
+Create a new Agent Pool for the agents which will be responsible for the VirtualBox Packer builds and the assign the new Agent to the new Agent Pool.
+
+## 5. Create a new Build in TeamCity
+
+In TeamCity Server create a new build and configure the Version Control Settings to download the Packer build configuration from the VCS repository. 
+
+Add one **Build Step: Command Line** to the build.
+
+![TeamCity screenshot: New Build](./images/teamcity_new_build.png)
+
+In the **Script content** field add the following:
+
+```shell
+#!/usr/bin/env bash
+/root/packer build -only=virtualbox-iso -var "headless=true" ./packer.json 
```

Comment:
headless=true is only applicable if you've structured your packer.json just so. I recommend we use an example json somewhere